### PR TITLE
Tweaks to installing jspm page

### PR DESCRIPTION
**Pull request by jackfranklin**

### CODE REVIEW

File: creating-a-project.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 ## Creating a Project
 
+First, make sure you've installed jspm. You can install jspm globally:
+
+```
+npm install --global jspm@0.17-beta.14
```
Comment (guybedford) [inline creating-a-project.md]: `npm install -g jspm@beta`?

File: creating-a-project.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+```
+npm install --save-dev jspm@0.17-beta.14
+```
+
+_Once jspm 0.17 is stable and released you'll be able to just run `npm install jspm`, but for the beta version you must specify the version._
+
+Note that if you install it locally the `jspm` executable will be placed within `./node_modules/.bin` and you'll have to execute it like so:
+
+```
+./node_modules/.bin/jspm
```
Comment (guybedford) [inline creating-a-project.md]: This isn't really advisable, and is always used by the global jspm anyway. So the best workflow is actually to install _both_ globally and locally.

**Comment by guybedford** [comment]
Thanks, I heard someone else had this catch as well, so it's great to know about these things!

File: creating-a-project.md (hunk 2 above)
Comment (jackfranklin) [inline creating-a-project.md]: Ah nice!

**Comment by jackfranklin** [comment]
@guybedford fixed now I think :)

**Comment by guybedford** [comment]
Awesome, thank you.

**Pull request merged**